### ISSUE TITLE
Can't search for aur packages

### ISSUE THREAD

**Issue by dlip** [manjaro/pamac#370]
Just done a fresh install of manjaro xfce 17.0.6 and pamac 6.2.1-1 with yaourt 1.9-1.
I can enable the aur option but when I try to search and select the AUR tab its always empty. Is there some way to get an error log?

One possibility is I'm using a corporate proxy, but yaourt install things fine from the command line.

**Comment by philmmanjaro** [comment]
**pamac** has it's own AUR engine, therefore your corporate proxy might be a reason it doesn't work. Did you test the same on a public or private network?

**Comment by dlip** [comment]
Yes its something to do with the corporate proxy, pamac is probably not using the company ca-certs since they decrypt ssl traffic. How to debug?

**Comment by guinux** [comment]
Pamac uses "http_proxy" and "https_proxy" environment variables. Are they defined ?

**Comment by dlip** [comment]
Yes I have them set in /etc/profile.d/proxy.sh and all other apps work.
That's why I suspect the ssl certificate.

**Comment by dlip** [comment]
Note: I have installed the certificate and can see it appears in /etc/ssl/certs/ca-certificates.crt and it works with curl

**Comment by lorenzosu** [comment]
Hi, I face some issues with pamac behind a company proxy even with http_proxy and https_proxy environment variables set. Most operations work but upgrade and AUR not. Some details below.

What works when running pamac-manager after setting the correct variables *and* making sure in the sudoers file these are also passed to sudo:
- Searching for packages
- Installing packages
- Refreshing database
- Refreshing mirror lsit

What doesn't work:
- Updates: pamac always says the system is up to date, but pacman will show upgrades actually exist
- Any AUR related operation (search, install, update). yaourt most of the time works with the above environment variables.

Why it would be nice to have the option for application-specific proxy configuration:
- In DE like XFCE (e.g. default in Manjaro, which in turn uses pamac as default graphical front-end for package management), there is no system-wide proxy configuration, so for many users tinkering with environment variables is a pain
- Many users are using a laptop and may be behind a proxy (typically at their company, client, etc.) and then move to somewhere where they are connecting directly. In this case setting system-wide environment variables (and removing them) is a pain. proxy on/off within the application is much more usable :)

My two cents.
Lorenzo.
